docs/design/features/byreflike-generics.md

```diff
@@ -0,0 +1,76 @@
+# Generics parameters of ByRefLike types
```

Comment:
Just to clarify: this is specifically about `Span<Span<byte>>` not `Span<ref byte>`?

Reply:
Correct. I believe that `Span<ref T>` would be slightly more complicated for all parties. This work is specifically about permitting `ref struct` types (ByRefLike) to be used as Generic parameters. Further details from the language side can be found at https://github.com/dotnet/csharplang/pull/5689.